redesign-bookthing/src/styles/layout/_grid.scss

```
@@ -0,0 +1,96 @@
+//homepage grid desktop
+.homepage-grid {
+  display: grid;
+  grid-template-areas:
+    "header content content"
+    "nav content content"
+    "footer hours location";
+  height: 100vh;
+  width: 100vw;
+  grid-template-columns: .3fr, 1fr, 1fr;
```

The commas here are breaking your code:
```suggestion
  grid-template-columns: .3fr 1fr 1fr;
```